### PR TITLE
[New] add RE:DOM v1.5.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by pakastin**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 - [x] Git repository (required):
 - [x] Official website (optional, not the repository):
 - [x] NPM package url (optional):
 - [x] GitHub / Bitbucket popularity (required):
   - Count of stars: 593
   - Count of watchers: 26
   - Count of forks: 24
 - [x] NPM download stats (optional):
   - Downloads in the last day: 53
   - Downloads in the last week: 152
   - Downloads in the last month: 990

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by pakastin** [comment]
Oh sorry, I thought they will get pulled from NPM..

File: ajax/libs/redom/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "RE:DOM",
+  "description": "Tiny UI library",
+  "homepage": "https://redom.js.org",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/pakastin/redom"
+  },
+  "author": "Juha Lindstedt",
+  "license": "ISC",
```
Comment (pvnr0082t) [inline ajax/libs/redom/package.json]: @pakastin I think the license is `"MIT"`. Ref: https://github.com/pakastin/redom/blob/master/LICENSE

File: ajax/libs/redom/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "client-side",
+    "dom"
+  ],
+  "filename": "redom.min.js",
+  "npmName": "redom",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "redom*.js"
```
Comment (nolsherry) [inline ajax/libs/redom/package.json]: @pakastin, I think `"redom*"` is better.

**Comment by pakastin** [comment]
Should be ok now!

**Comment by pakastin** [comment]
Now added subdirectory!

**Comment by pakastin** [comment]
> Now CDNJS supports adding a library by a single package.json, you just need to add a valid CDNJS package.json with npm/git auto-update config, and remove its version field in package.json, then we'll handle the remaining works.
So this is not anymore supported?

**Comment by pakastin** [comment]
I'm confused. Last time I added a library, it was so much simpler:
https://github.com/cdnjs/cdnjs/pull/7725/files

**Comment by pakastin** [comment]
Referring to #7793

**Comment by pvnr0082t** [comment]
@pakastin Sorry for making you confused. If you add a lib via single package.json, you don't need to commit any files else. For now, you committed the latest version(v1.5.0) and auto-update will update to v1.4.0 and add version field `"1.4.0"` in package.json. It will be wrong.

**Comment by pakastin** [comment]
1.5.0 is the latest version.. 🤔

**Comment by pakastin** [comment]
Well, I added the `"version"` now..

**Comment by pvnr0082t** [comment]
yeah, v1.5.0 is the latest version, but auto-update will not fetch the existing version in cdnjs. It will stop at v1.4.0.

**Comment by pakastin** [comment]
I dropped completely. What should I do?

**Comment by pvnr0082t** [comment]
I think it looks good now. Let's wait for CI test. Thanks.

**Comment by pakastin** [comment]
ok, great 👍

**Comment by pvnr0082t** [comment]
LGTM
ping @kennynaoh @nolsherry

**Comment by kennynaoh** [comment]
LGTM

**Comment by nolsherry** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 the title is not good enough.

**Comment by pakastin** [comment]
It wasn't there on that version. This process has taken so long I've pushed new versions since.

**Comment by pakastin** [comment]
What happened to that one-file process (only package.json and that's it)?

**Comment by PeterDaveHello** [comment]
Fair enough, will merge this right now, thanks @pakastin

**Comment by pakastin** [comment]
Great, thanks! Will it auto-update or did I do something wrong this time?

**Comment by PeterDaveHello** [comment]
@pakastin It'll auto update by default, since there are many other versions of it, I'll manually import them which is faster.

**Comment by pakastin** [comment]
Ok, thank you! 👍😎👍

**Comment by PeterDaveHello** [comment]
It's online now: https://cdnjs.com/libraries/redom/1.6.3

**Pull request merged**